src/asciidoc/docs/develop/editor.adoc

```diff
@@ -1,123 +1,156 @@
-You can use the Koji editor to create application and game templates.
-The editor contains all of your project's code, Visual Customization Controls (VCCs) generated from the project's code, interfaces for publishing and deployment management, and interfaces to other production tools such as the Koji database.
 
 == Overview
 
-The Koji editor is divided into these sections:
+To create a new Koji, you must set up a project and open it in the Koji Code Editor.
+[.singlespaced]
+. Create a project.
++
+include::{includespath}/steps-codeeditor.adoc[tag=createproject-details-collapsible]
+. {blank}
++
+include::{includespath}/steps-codeeditor.adoc[tag=open-in-code-editor]
 
-* *Navigation pane* – Lists menu items and source code.
-* *Editing pane* – Displays open files.
-Supports multiple tabs.
-* *Preview pane* – Displays a live preview of the template and provides other testing and debugging tools.
-* *Terminal pane* – Provides command line access to the project's services.
-Supports multiple tabs and can be collapsed.
+To update an existing Koji, simply open the associated project in the Koji Code Editor.
+[.singlespaced]
+. {blank}
++
+include::{includespath}/steps-codeeditor.adoc[tag=openproject]
 
-image::Koji-editor.png[Screenshot of the Koji editor process]
+. {blank}
++
+include::{includespath}/steps-codeeditor.adoc[tag=open-in-code-editor]
 
-== Koji editor code hosting and source control
 
-Each Koji project is backed by a Git repository.
-When you remix a project, Koji clones that project's Git repository to your account and opens it in the editor.
-When you publish your application from the editor, Koji commits your changes and pushes to master.
-This push triggers a deploy.
-You can think of the state of your Koji editor in the same way as the state of a Git repository on your computer -- that is, it is a working HEAD of your repository.
-This means that if you make changes to your project from outside the Koji editor, you must `git pull` those changes from within the Koji editor to see them.
+include::{includespath}/images-annotated.adoc[tag=editorsections]
 
-If you would like to use a remote editor, you must generate a Git access token.
-You can use this access token in place of a password when cloning your repositories.
-To find the remote URL of your project's Git repository, go to *Advanced > Repository Info* in the Koji editor.
 
-For in-depth walkthroughs of working outside the Koji editor, see <<use-git#>>.
+== Koji source control
 
-== Previews
+Each Koji project is stored in a Git repository in the cloud.
 
-You can select the type of preview to display in the preview pane.
-To learn more about using previews, see <<testing-templates#_testing_in_the_koji_editor,Testing in the Koji editor>>.
-The following types are available.
+When you fork an existing project or scaffold:
+[.singlespaced]
+* The Git repository of that project/scaffold is copied to your account.
+* A new project is created and opened in the Code Editor.
```

Comment:
I don't think this is true anymore. Now, you can choose to open it in the code editor, but the project details page is what opens (as describe in your procedure earlier in the page. I would just write this as a sentence and remove the second bullet so we don't have to maintain this info in two places on the page. The info in the first bullet is most relevant for this section.

Reply:
Fixed.